### DISCUSSION TITLE
error 404 in displaying my website

### DISCUSSION THREAD

**Discussion by apooja1** [community/community#120587] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hello there, 
I am relatively new to GitHub. I created a repository to host my website apooja1.github.io, which I changed to private a couple of weeks ago. Starting then, I received error 404 when I visited my website. So, I changed it to public again. But that didn't solve the problem. would appreciate your help in solving this 404 error. thanks!

**Comment by Mazp17** [comment]
Good afternoon, I recommend you deploy the page again. 
If the problem persists, you can open a new discussion.
